tasks/replica.yml

```diff
@@ -1,6 +1,6 @@
 - name: Stop PostgreSQL
   service:
-    name: "{{ pgsql_version | pgsql_service_string }}"
+    name: postgresql
```

Comment:
The service name on RHEL7 is `postgresql-9.6`. This service name seems incorrect.

```
# systemctl stop postgresql
Failed to stop postgresql.service: Unit postgresql.service not loaded.
```